cmake/macros/SundialsAddLibrary.cmake

```diff
@@ -150,7 +150,7 @@ macro(sundials_add_library target)
     # --------------------------------------------------------------------------
 
     # create the target for the object library
-    add_library(${obj_target} OBJECT ${sources})
+    add_library(${obj_target} OBJECT ${sources} ${sundials_add_library_UNPARSED_ARGUMENTS})
```

Comment:
Commenting here in case it's not clear below, what are the extra inputs that are needed?

Reply:
I might have missed it, but I didn't see where this was needed.

Reply:
I ended up not using it, but if you wanted to pass extra inputs to go to `add_library` from `sundials_add_library` this would be needed.